libs/fleet-mcp/src/fleet_mcp/coder/discovery.py

```diff
@@ -93,3 +93,59 @@ async def is_valid_fleet_mcp_project(
     """
     valid_names = await get_valid_fleet_mcp_project_names(coder_client)
     return project_name in valid_names
+
+
+async def resolve_project_identifier_to_template_name(
```

Comment:
Rename this to `resolve_project_name_to_template`. This method accepts a project name like "Setup" or "DataOne" and will return a Template (or raise a validation error). Don't worry about backwards compatibility, only focus on supporting project names!